File: src/frontend/src/services/openapi/models/Component.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { Translation } from './Translation';

export type Component = {
    readonly id?: string;
    description?: string | null;
    name?: string | null;
    isLock?: boolean;
    translations?: Array<Translation> | null;
}

File: src/backend/Models/Component.cs
using System;
using System.Collections.Generic;

namespace Sleet.Models
{
    public class Component {
        
        public Guid Id { get; set; }
        public string Description { get; set; }
        public string Name { get; set; }
        public bool IsLock { get; set; }
        public List<Translation> Translations { get; set; }
    }
}
